runtime/redirector/redirector.c

```diff
@@ -693,22 +699,32 @@ chooseJVM(JavaVMInitArgs *args, char *retBuffer, size_t bufferLength)
 		}
 		fprintf(stdout, "does not exist.\n");
 
+#if defined(OMR_GC_COMPRESSED_POINTERS) && defined(OMR_GC_FULL_POINTERS)
+		fprintf(stdout,
+				"This JVM package includes both the '-Xcompressedrefs' and the '-Xnocompressedrefs' "
+				"configurations, however the VM directory could not be found. Please download the latest "
+				"JVM package or build with the most recent changes and run the JVM again.\n"
+		);
+#else
 		/* direct user to OpenJ9 build configurations to properly generate the requested build. */
 		if (0 == strcmp(OPENJ9_NOCR_JVM_DIR, basePointer)) {
 			fprintf(stdout,
 					"This JVM package only includes the '-Xcompressedrefs' configuration. Please run "
 					"the VM without specifying the '-Xnocompressedrefs' option or by specifying the "
 					"'-Xcompressedrefs' option.\nTo compile the other configuration, please run configure "
-					"with '--with-noncompressedrefs.\n"
+					"with '--with-noncompressedrefs', or run configure with '--with-mixedrefs=static' or "
```

Comment:
This help text is questionable - we won't neccessarily be using configure.

Reply:
What's the other method? Should the configure text be removed, or should we include the corresponding help text for the other method?

Reply:
I think it should just be removed - it's an essentially impossible situation.